src/anomalib/deploy/inferencers/torch_inferencer.py

```diff
@@ -181,14 +181,14 @@ def predict(
         if isinstance(image, str | Path):
             image = read_image(image, as_tensor=True)
 
-        metadata["image_shape"] = image.shape[-2:]
+        metadata["image_shape"] = image.transpose(2,0,1).shape[-2:]
```

Comment:
This is to pass the code quality checks.
```suggestion
        metadata["image_shape"] = image.transpose(2, 0, 1).shape[-2:]
```